runner/src/load_generator/utils/generator_utils.rs

```diff
@@ -0,0 +1,105 @@
+use crate::scenario::ceramic::CeramicClient;
+use crate::scenario::ceramic::Credentials;
+use anyhow::Result;
+use ceramic_http_client::CeramicHttpClient;
+use reqwest::Client;
+use std::time::Duration;
+
+use super::ceramic_models_utils::CeramicModelUser;
+
+pub static HTTP_TIMEOUT: Duration = Duration::from_secs(5);
+pub static HTTP_POOL_MAX_IDLE_PER_HOST: usize = 300;
```

Comment:
these could be `const` but it doesn't matter in practice. `const` is inserted into the calling location at compile time, so it's more performant, `static` is one shared memory location. you can't use const for shared data as it won't do what you want, and not all functions are supported but this one should be fine